src/components/KBreadcrumbs/KBreadcrumbs.vue

```diff
@@ -98,66 +112,71 @@ export default {
 @import '@/styles/functions';
 
 .k-breadcrumbs {
-  border-radius: 4px;
+  border-radius: var(--kui-border-radius-4, $kui-border-radius-4);
   display: flex;
   -ms-flex-wrap: wrap;
   flex-wrap: wrap;
-  font-size: 15px;
-  font-weight: 600 !important;
-  line-height: 24px !important;
+  font-size: var(--kui-font-size-30, $kui-font-size-30);
+  font-weight: var(--kui-font-weight-medium, $kui-font-weight-medium);
+  line-height: var(--kui-line-height-40, $kui-line-height-40);
   list-style: none;
-  margin-bottom: 16px;
-  padding: 0;
+  margin-bottom: var(--kui-space-60, $kui-space-60);
+  padding: var(--kui-space-0, $kui-space-0);
 
   .k-breadcrumbs-item {
     .k-breadcrumb-divider,
     .k-breadcrumb-icon {
       align-self: center;
-      color: var(--grey-500, color(grey-500));
+      color: var(--kui-color-text-decorative, $kui-color-text-decorative);
```

Comment:
```suggestion
      color: var(--grey-500, var(--kui-color-text-decorative, $kui-color-text-decorative));
```

issue 1: you need to keep the old CSS var and add new tokens as fallback like so. This and many other places in this PR.

issue 2: `kui-color-text-decorative` is alias for `aqua.50` color token, which doesn't translate to old `grey-500` that was used here. Did you mean to make this change?